packages/docs/docs/api-reference/form-props.md

```diff
@@ -83,6 +83,9 @@ Optional enumerated flag controlling how array minItems are populated, defaultin
 | `requiredOnly` | Ignore `minItems` on a field when calculating defaults unless the field is required.                                                               |
 | `never`        | Ignore `minItems` on a field when calculating defaults for required and non-required. Value will set only if defined `default` and from `formData` |
 
+#### `arrayMinItems.computeSkipPopulate`
+A function that determines whether to skip populating the array with default values based on the provided validator, schema, and root schema. If the function returns `true`, the array will not be populated with default values. If the function returns `false`, the array will be populated with default values according to the `populate` option.
```

Comment:
I think providing the expected signature of this method and perhaps a minimal `Typescript` example would be helpful?

Reply:
I added an example.